frontend/assets/style/components/_buttons.scss

```diff
@@ -212,7 +212,7 @@ button.link {
   color: $primary_0;
   font-weight: 600;
   display: inline-block;
-  line-height: inherit;
+  line-height: 1.4; // better underline visually
```

Comment:
@pieer can you verify this change? Does it make sense to not use a variable here?

Reply:
Inherit is not a variable. All good :)

Reply:
OK, not sure I understand since `inherit` is being replaced with `1.4`, but assume it's OK to you

Reply:
inherit is receives the value from the body, in this case `1.5`. That causes the link border-bottom to be too far from the text. So I manually decreased it a little to look better. Here's an image for you:

![image](https://user-images.githubusercontent.com/14869087/68476602-09d0a100-0223-11ea-9432-9f4eeef9be57.png)

The first paragraph has `line-height: inherit;` (1.5) and the second `1.4`.